octobot_trading/exchanges/data/exchange_personal_data.py

```diff
@@ -102,14 +100,15 @@ async def handle_portfolio_profitability_update(self, balance, mark_price, symbo
     async def handle_order_update_from_raw(self, symbol, order_id, raw_order, should_notify: bool = True) -> bool:
         try:
             changed: bool = self.orders_manager.upsert_order_from_raw(order_id, raw_order)
-            if should_notify:
-                await get_chan(ORDERS_CHANNEL, self.exchange_manager.id).get_internal_producer() \
-                    .send(cryptocurrency=self.exchange_manager.exchange.get_pair_cryptocurrency(symbol),
-                          symbol=symbol,
-                          order=raw_order,
-                          is_from_bot=True,
-                          is_closed=False,
-                          is_updated=changed)
+
+            if changed:
+                if await self._handle_order_post_update(self.orders_manager.get_order(order_id)):
+                    # when the order is removed from orders_manager
+                    return changed
```

Comment:
No notification when order is closed ?

Reply:
if close it calls `handle_closed_order_update` that manages the close process with the push in Orders and in Trade channels. If a notifier is following one of these two channels there will be a notification